sdk/rust/sdk/src/chains/terra.rs

```diff
@@ -0,0 +1,63 @@
+use cosmwasm_std::{
+    to_binary,
+    Addr,
+    Binary,
+    CosmosMsg,
+    DepsMut,
+    Env,
+    QueryRequest,
+    StdResult,
+    WasmMsg,
+    WasmQuery,
+};
+use serde::Serialize;
+
+use bridge::msg::{
+    ExecuteMsg,
+    QueryMsg,
+};
+use bridge::state::ParsedVAA;
+
+/// Export Core Mainnet Contract Address
+#[cfg(feature = "mainnet")]
+pub fn id() -> Addr {
+    Addr::unchecked("terra1dq03ugtd40zu9hcgdzrsq6z2z4hwhc9tqk2uy5")
+}
+
+/// Export Core Devnet Contract Address
+#[cfg(feature = "devnet")]
+pub fn id() -> Addr {
+    Addr::unchecked("terra1pd65m0q9tl3v8znnz5f5ltsfegyzah7g42cx5v")
+}
+
+pub fn post_message<T>(wormhole: Addr, nonce: u32, message: &T) -> StdResult<CosmosMsg>
```

Comment:
Why do these functions take the wormhole address as an argument, and not just use `id()` internally?